docs-parts/intro/Releases_lang1.rst

```diff
@@ -15,6 +15,7 @@
 * Bugfix - Fix assertion error when performing a union into a join (#930) PR #967
 * Bugfix - Fix regression issue with `DISTINCT` clause and `GROUP_BY` (#914) PR #963
 * Update `~jobs.error_stack` from blob to mediumblob to allow error stacks >64kB in jobs (#984) PR #986
+* Bugfix - Fix error when altering a part table that uses the "master" keyword (#936) PR #991
```

Comment:
Would you repull from `upstream` and fix any merge conflicts? Once you do, you will notice that this doc doesn't exist anymore and we only need to update this in one place (`CHANGELOG.md`).